src/coreclr/jit/valuenum.cpp

```diff
@@ -11165,6 +11165,7 @@ void Compiler::fgValueNumberTree(GenTree* tree)
 
             // These do not represent values.
             case GT_NO_OP:
+            case GT_NOP:
```

Comment:
There is code above:

https://github.com/dotnet/runtime/blob/09eebddbbb1e75a6ab5ecd9dc85520f0ad9b93cf/src/coreclr/jit/valuenum.cpp#L11391-L11394

which should be deleted. I don't think the "actually nullary" case there is reachable now.